lib/analyzer.h

```diff
@@ -188,10 +188,15 @@ struct Analyzer {
     virtual bool invalid() const {
         return false;
     }
+    int countBranch() {
+        return ++mBranchCount;
+    }
     virtual ~Analyzer() = default;
     Analyzer(const Analyzer&) = default;
 protected:
     Analyzer() = default;
+private:
+    int mBranchCount = 0;
```

Comment:
This should go into the `ForwardTraversal` class instead.

Reply:
okay. Fixed by c51745885f2f5c1908f5d93404caa2bd8952ec87